demo/site/src/app/[domain]/[language]/[[...path]]/page.tsx

```diff
@@ -1,3 +1,5 @@
+export const dynamic = "error";
```

Comment:
I've also thought of this. We should add it in the Starter.

Reply:
yes, all of this PR